app/domain/domain.go

```diff
@@ -1 +1,40 @@
 package domain
+
+import (
+	"math"
+	"time"
+
+	"github.com/domainr/whois"
+	whoisparser "github.com/likexian/whois-parser-go"
+)
+
+// GetDomainInfo makes request for domain info
+func GetDomainInfo(domain string) (whoisparser.WhoisInfo, error) {
+	request, err := whois.NewRequest(domain)
+
+	if err != nil {
+		return whoisparser.WhoisInfo{}, err
+	}
+
+	response, err := whois.DefaultClient.Fetch(request)
+
+	if err != nil {
+		return whoisparser.WhoisInfo{}, err
+	}
+
+	domainInfo, err := whoisparser.Parse(string(response.Body))
+
+	if err != nil {
+		return whoisparser.WhoisInfo{}, err
+	}
+
+	return domainInfo, nil
+}
+
+// GetDaysFromCreation get days from creation date
+func GetDaysFromCreation(creationDate time.Time) int {
```

Comment:
I would rename it to something like `GetDaysFromDate` as `created` is unnecessary here, there is nothing restricting you from giving any date and receiving days from it.

Reply:
Also, I would move it to separate file like `helpers.go` as it's not tied to domains directly.